src/wwwroot/js/genpage/gentab/presets.js

```diff
@@ -292,16 +293,91 @@ function editPreset(preset) {
     fixPresetParamClickables();
 }
 
+let presetSortByElement;
+let presetSortReverseElement;
+let presetSortBy = localStorage.getItem('preset_list_sort_by') || 'Default';
+let presetSortReverse = localStorage.getItem('preset_list_sort_reverse') === 'true';
+
+
+function onPresetSortByChanged(ev) {
+    localStorage.setItem('preset_list_sort_by', presetSortBy = ev.target.value);
```

Comment:
there's a lot of odd code format choices in here

Reply:
can you provide any specifics on what you find odd about the format?

Reply:
ok I've gone through and cleaned up the spacing, added comments, and (even though it feels really weird) removed the usages of `const` and `===` in favor of the looser `let` and `==` syntax.

Let me know if it's still out of line with your JS usage preferences.